crates/rpc/ipc/Cargo.toml

```diff
@@ -30,6 +29,11 @@ tracing.workspace = true
 bytes.workspace = true
 thiserror.workspace = true
 futures-util = "0.3.30"
+interprocess = { git = "https://github.com/kotauskas/interprocess", rev = "5023981c", features = ["tokio"] }
```

Comment:
why a specific commit over the latest release? we'd like to avoid git deps where possible

Reply:
Done.